x-pack/qa/sql/src/main/resources/geo/geosql.csv-spec

```diff
@@ -18,28 +18,58 @@ DESCRIBE "geo";
+Amsterdam      |52.347557, 4.850312    |point (4.850312 52.347557)    |Europe
+Berlin         |52.486701, 13.390889   |point (13.390889 52.486701)   |Europe
+Chicago        |41.888783, -87.637874  |point (-87.637874 41.888783)  |Americas
+Hong Kong      |22.281397, 114.183925  |point (114.183925 22.281397)  |Asia
+London         |51.510871, -0.121672   |point (-0.121672 51.510871)   |Europe
+Mountain View  |37.386483, -122.083843 |point (-122.083843 37.386483) |Americas
+Munich         |48.146321, 11.537505   |point (11.537505 48.146321)   |Europe
+New York       |40.745171, -73.990027  |point (-73.990027 40.745171)  |Americas
+Paris          |48.845538, 2.351773    |point (2.351773 48.845538)    |Europe
+Phoenix        |33.376242, -111.973505 |point (-111.973505 33.376242) |Americas
+San Francisco  |37.789541, -122.394228 |point (-122.394228 37.789541) |Americas
+Seoul          |37.509132, 127.060851  |point (127.060851 37.509132)  |Asia
+Singapore      |1.295868, 103.855535   |point (103.855535 1.295868)   |Asia
+Sydney         |-33.863385, 151.208629 |point (151.208629 -33.863385) |Asia
+Tokyo          |35.669616, 139.76402225|point (139.76402225 35.669616)|Asia
+;
+
+// TODO: Both shape and location contain the same data for now, we should change it later to make things more interesting
+selectAllPointsAsWKT
+SELECT city, ST_ASWKT(location) location, ST_ASWKT(shape) shape, region FROM "geo" ORDER BY "city";
+
+     city:s    |     location:s               |        shape:s               |  region:s
+Amsterdam      |point (4.850312 52.347557)    |point (4.850312 52.347557)    |Europe
+Berlin         |point (13.390889 52.486701)   |point (13.390889 52.486701)   |Europe
+Chicago        |point (-87.637874 41.888783)  |point (-87.637874 41.888783)  |Americas
+Hong Kong      |point (114.183925 22.281397)  |point (114.183925 22.281397)  |Asia
+London         |point (-0.121672 51.510871)   |point (-0.121672 51.510871)   |Europe
+Mountain View  |point (-122.083843 37.386483) |point (-122.083843 37.386483) |Americas
+Munich         |point (11.537505 48.146321)   |point (11.537505 48.146321)   |Europe
+New York       |point (-73.990027 40.745171)  |point (-73.990027 40.745171)  |Americas
+Paris          |point (2.351773 48.845538)    |point (2.351773 48.845538)    |Europe
+Phoenix        |point (-111.973505 33.376242) |point (-111.973505 33.376242) |Americas
+San Francisco  |point (-122.394228 37.789541) |point (-122.394228 37.789541) |Americas
+Seoul          |point (127.060851 37.509132)  |point (127.060851 37.509132)  |Asia
+Singapore      |point (103.855535 1.295868)   |point (103.855535 1.295868)   |Asia
+Sydney         |point (151.208629 -33.863385) |point (151.208629 -33.863385) |Asia
+Tokyo          |point (139.76402225 35.669616)|point (139.76402225 35.669616)|Asia
+;
+
+selectWithAsWKTInWhere
+SELECT city, ST_ASWKT(location) location, region FROM "geo" WHERE LOCATE('114', ST_ASWKT(location)) > 0 ORDER BY "city";
```

Comment:
Can you add more tests that use the new functions in ORDER BY, GROUP BY and HAVING? This should test the Painless script in various setups. Same goes for tests using these functions inside other functions or the input of these functions to come from other functions, useful for testing folding.